<launch>
 <arg name="mag_declination_inertial" default="4.260"/>


 <node pkg="turtlebot_polaris" type="turtlebot_polaris_node.py" name="controller" output="screen">
  <param name="mag_declination_inertial" value="$(arg mag_declination_inertial)"/>


  <param name="loop_freq" value="30.0"/>
  <param name="max_ang_speed_rad_s" value="1.0"/>
  <param name="kp" value="1.0"/>
  <param name="ki" value="0.0"/>
 </node>

 </launch>